tests/cassettes/TestCube.test_microcerts_json.yaml

```diff
@@ -1,219 +0,0 @@
-interactions:
-- request:
-    body: null
```

Comment:
Regenerating this cassette can be a pain, it was manually tampered so it can pass the mocked test.